### DISCUSSION TITLE
Astonishing realistic nice Mars surface simulation!

### DISCUSSION THREAD

**Discussion by isenberg** [Stellarium/stellarium#3058] [Show and tell]
The Mars surface simulation regarding sky color, sky brightness and surface brightness is astonishingly **precisely matching the real observations seen by the Navcam pair on Perseverance Rover**! **Great work!** Stellarium actually helped me here to verify my UTC clock display I recently added and  noticed that the image meta data as spacecraft clock in the NASA PDS raw images is off by more than a minute, at least from what I see about the zero point of the Mars2020 mission clock! Check out my daily automated Perseverance Rover album: https://areo.info/mars20 

Recently found the Mars moons and one orbiter in the Navcam images and could verify the moons with Stellarium. For MGS I used https://eyes.nasa.gov to exclude other more recent orbiters and then Cosmographia and a downloaded SPICE file both from https://naif.jpl.nasa.gov .
 
Phobos:
https://twitter.com/areoinfo/status/1625025823005900800
https://twitter.com/areoinfo/status/1605071897145643008

Deimos:
https://twitter.com/areoinfo/status/1608231002245328896

the since 2007 defunct Mars Global Surveyor (MGS):
https://twitter.com/areoinfo/status/1617066814059737088

**Comment by gzotti** [comment]
Be sure about UTC vs. DT! Also when you observe from any significant distance, light time comes into play.

**Reply by isenberg** [reply]
The UTC here is universe-wide synchronous, at least that's how I see it in this case and it matches the observations. As Phobos is much nearer to Mars than our Moon is to Earth, there is no noticeable light propagation delay.

But you are right about always considering light delay. I once was confused about a 40min delay between the universe-wide UTC in a simulation and the position of Jupiter moons :)

**Reply by gzotti** [reply]
UTC is not even uniformly growing. If you see a ~>1min difference between self-computed UTC results and results given elsewhere, I would triple-check if these other results not rather give DT dates.

**Reply by isenberg** [reply]
I just checked against the stars, they never lie :)

If anyone else likes to double check: The timestamp the image was taken is in the filename of each Mars Perseverance image file.

Example for the first image of the Phobos series:
NLF_0701_0729149912_181ECM_N0330366NCAM00516_01_295J.png

The 0729149912 is the spacecraft clock in seconds since 2000-01-01 11:58:55.816 UTC according to https://naif.jpl.nasa.gov/pub/naif/MARS2020/kernels/sclk/M2020_168_SCLKSCET.00006.tsc

And if you add that number of seconds to 2000-01-01 11:58:55.816 UTC you'll arrive at Wed Feb  8 17:37:27 UTC 2023 which matches the actual Phobos location in the Navcam image and Stellarium simulation UTC clock.

But interesting you won't get that UTC time from the meta data when the raw PDS IMG file will be released in about 6 months on
https://pds-imaging.jpl.nasa.gov/data/mars2020/mars2020_navcam_ops_raw

At least that was my result when I tried that for the Deimos observation on Sol 173 where the raw data is already release. That explains why the first images I published are a bit off on https://twitter.com/areoinfo/status/1608231002245328896 .

**Comment by 10110111** [comment]
> regarding sky color, sky brightness

But Stellarium doesn't simulate Mars' sky in any way, just uses the skylight model for the Earth... And the photos from Perseverance that you show in your tweets seem to be color-manipulated, because the Martian sky is unlikely to ever look blue (except for the solar aureole part, which is usually violet). Maybe only at super-clear weather, when there's almost no dust in the air, but then the sky would be really dim.

**Reply by isenberg** [reply]
Well, it just works currently, no need to change anything. For those who want some hostile-looking low contrast red light in the simulation you could consider to add a button. Maybe name it "Dust Storm Vision". But I really recommend to keep the current implementation available.

Here some recent PDFs about applying color space transformation from the camera raw image color space to sRGB as I do it for all images on my https://areo.info/mars20: https://www.researchgate.net/profile/Holger-Isenberg/research
